docs/config.yaml

```
@@ -38,8 +38,8 @@ streams:
       type: production
       # OPTIONAL: override cosa image to use for this stream
       cosa_image: "quay.io/jlebon/coreos-assembler:stable"
-      # OPTIONAL: additional artifacts to build for this stream
-      additional_artifacts:
+      # OPTIONAL: skip artifacts for this stream
+      skip_artifacts:
         aarch64:
           - azure
         ppc64le:
```

Let's update the example to make more sense based on the rest of the file.

```suggestion
      # OPTIONAL: skip artifacts for this stream
      skip_artifacts:
        aarch64:
          - openstack
        x86_64:
          - vmware
```